packages/core/src/use-params.ts

```diff
@@ -97,6 +97,11 @@ export function useParam(
 ): undefined | number | string | Array<string | undefined> {
   const params = useParams()
   const rawValue = params[key]
+  const exists = params.hasOwnProperty(key)
```

Comment:
Nit, what do you think about rename exists to something like `keyExists`. This is not a blocking change, but it helps with readability.

Reply:
I could make this change if you want, but I'll wait for the resolution of the above discussion first

Reply:
Going to mark this as resolved :)